src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int totalSupply = 0;
+        int totalBuy = 0;
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] parts = line.split(",");
+                if (parts.length < 2) {
+                    continue;
+                }
+                String operation = parts[0].trim();
+                int amount = Integer.parseInt(parts[1].trim());
+
+                if (operation.equals("supply")) {
+                    totalSupply += amount;
+                } else if (operation.equals("buy")) {
+                    totalBuy += amount;
+                }
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't correctly read data from file " + fromFileName, e);
+        }
```

Comment:
Consider using a separate method for reading from the file and processing the data, which will return the calculated values. This will make the code more modular and easier to test.